src/test/java/picard/analysis/CollectMultipleMetricsTest.java

```diff
@@ -335,8 +335,10 @@ void setupBuilder() throws IOException {
         final String flowCellBarcode = "TESTBARCODE";
 
         tempSamFile = File.createTempFile("CollectGcBias", ".bam", TEST_DIR);
+        final File tempSamIndex = new File(tempSamFile.getAbsolutePath().replace("bam", "bai"));
```

Comment:
Is the index a valid index? If you run ValidateSamFile on the bam/bai pair, do the files validate?